requirements/core.txt

```diff
@@ -1,12 +1,12 @@
-numpy==1.19.5
-xarray==0.18.2
-scipy==1.6.2
-statsmodels==0.12.2
-matplotlib==3.4.1
-pandas==1.2.3
-ipython==7.22.0
-tqdm==4.60.0
-holidays==0.11.1
-scikit-learn==0.24.1
-joblib==1.0.1
-requests==2.25.1
\ No newline at end of file
+numpy>=1.19.0
+xarray>=0.18.0
+scipy>=1.6.0
+statsmodels>=0.12.0
+matplotlib>=3.4.0
+pandas>=1.2.0,<1.3.0
```

Comment:
any reason why this is upper bounded?

Reply:
pandas 1.3.0 broke the tests, something to be investigated by someone more familiar with the codebase :)